test/nn.jl

```diff
@@ -59,7 +59,8 @@ end
 end
 
 
-for (rnn_fun, post_proc_outputs) in ((nn.dynamic_rnn, identity), (nn.rnn, last))
```

Comment:
This file also lets use `@test_broken`?
It is a bit harder as it will need a conditional  to check the `rnn_fun`